vllm/engine/async_llm_engine.py

```diff
@@ -924,6 +925,14 @@ async def get_model_config(self) -> ModelConfig:
         else:
             return self.engine.get_model_config()
 
+    async def get_parallel_config(self) -> ParallelConfig:
```

Comment:
can these new methods go into the `VLLMBackend` protocol as well?

Reply:
I dont think they should be in the `protocol` because the Protocol does not have to implement these + most of the time the Protocol will not implement these

Reply:
ah yeah I see these are only on the AsyncLLMEngine, 🌶️